File: Patterns/python/pattern15.py
# n = 4

# A
# B C
# C D E
# D E F G

n = int(input())
row = 1
startChr = 65
while row <= n:
    col = 1
    val = startChr
    while col <= row:
        print(chr(val), end="")
        col += 1
        val += 1
    row += 1
    startChr += 1
    print("")

File: Patterns/js/pattern15.js
/*
n = 4

A
B C
C D E
D E F G

*/

function pattern(n) {
  let ans = "";
  let row = 1;
  let startingCharacter = 65;

  while (row <= n) {
    let col = 1;
    let val = startingCharacter;
    while (col <= row) {
      ans += String.fromCharCode(val) + " ";
      col++;
      val++;
    }

    startingCharacter++;
    row++;
    ans += "\n";
  }

  console.log(ans);
}

pattern(4);
